### PR TITLE
[RFC] vim-patch:7.4.{1053,1071}

### PR DESCRIPTION
**Pull request by jamessan**
**vim-patch:7.4.1053**
Problem:    Insufficient testing for quickfix commands.
Solution:   Add a new style quickfix test. (Yegappan Lakshmanan)

https://github.com/vim/vim/commit/da59dd5da6440c3410866ed61ce169a2012ba5bd

**vim-patch:7.4.1071**
Problem:    New style tests are executed in arbitrary order.
Solution:   Sort the test function names. (Hirohito Higashi)
            Fix the quickfix test that depended on the order.

https://github.com/vim/vim/commit/cfc0a350a9fa04f1b0cfa1ba31fbd2847376513f

### CODE REVIEW

File: test/functional/legacy/quickfix_spec.lua [hunk 1]
```diff
@@ -2,16 +2,294 @@
 
 local helpers = require('test.functional.helpers')
 local source, clear = helpers.source, helpers.clear
+local execute, nvim, call = helpers.execute, helpers.meths, helpers.call
```
Comment (KillTheMule) [inline test/functional/legacy/quickfix_spec.lua]: Execute isn't used in this file.

File: test/functional/legacy/quickfix_spec.lua [hunk 2]
```diff
@@ -2,16 +2,294 @@
...
+
+local function expected_empty()
```
Comment (KillTheMule) [inline test/functional/legacy/quickfix_spec.lua]: This function isn't used. You can tell because you did not provide `eq` :)

**Comment by KillTheMule** [comment]
I've added line comments, but `make testlint` should provide usefull output by now.

File: test/functional/legacy/quickfix_spec.lua (hunk 1 above)
Comment (jamessan) [inline test/functional/legacy/quickfix_spec.lua]: Yeah, that's left over from when I was fiddling around with some failures.

File: test/functional/legacy/quickfix_spec.lua (hunk 2 above)
Comment (jamessan) [inline test/functional/legacy/quickfix_spec.lua]: It's supposed to be, but I removed the calls when I was trying to figure out a hang.

**Comment by jamessan** [comment]
@jszakmeister Is the OSX failure here a sporadic issue or something that needs to be looked at on that system?

**Pull request merged**